src/devices/video/mc6845.cpp

```diff
@@ -932,7 +932,7 @@ void mc6845_device::handle_line_timer()
 			m_cursor_x = m_cursor_addr - m_line_address;
 
 			/* Schedule CURSOR ON signal */
-			m_cur_on_timer->adjust(cclks_to_attotime(m_cursor_x));
+			m_cursor_timer->adjust(cclks_to_attotime(m_cursor_x));
```

Comment:
I think this isn’t going to turn the cursor on.  You’ve changed it to use the parameter as the cursor value, and it will default to zero here, so this will effectively keep the cursor off.

Reply:
Applied the same fix as `mc6847` and `crtc_ega`.